libs/sdk-ui-kit/src/Tabs/Tabs.tsx

```diff
@@ -89,6 +89,15 @@ export class Tabs extends Component<ITabsProps, ITabsState> {
                 onClick={() => {
                     this.selectTab(tab);
                 }}
+                onKeyDown={(event) => {
+                    if (event.key === "Enter" || event.key === " ") {
```

Comment:
This condition repeats on many places and is bit cryptic, mainly the key with value " ".  Maybe you could create some sort of helper that will be shared? `shouldProceedWithKeyboardAction(e)`? There's probably a better name.

Reply:
extracted as isActionKey